cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/streaming/BufferingInputStream.java

```diff
@@ -30,29 +30,28 @@
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicLong;
 
-import org.apache.cassandra.spark.data.SSTable;
-import org.apache.cassandra.spark.stats.Stats;
+import org.apache.cassandra.spark.stats.IStats;
 import org.apache.cassandra.spark.utils.ThrowableUtils;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * The InputStream into the CompactionIterator needs to be a blocking {@link java.io.InputStream},
  * but we don't want to block on network calls, or buffer too much data in memory otherwise we will hit OOMs for large Data.db files.
  * <p>
- * This helper class uses the {@link SSTableSource} implementation provided to asynchronously read
+ * This helper class uses the {@link CassandraFileSource} implementation provided to asynchronously read
  * the T bytes on-demand, managing flow control if not ready for more bytes and buffering enough without reading entirely into memory.
  * <p>
- * The generic {@link SSTableSource} allows users to pass in their own implementations to read from any source.
+ * The generic {@link CassandraFileSource} allows users to pass in their own implementations to read from any source.
  * <p>
  * This enables the Bulk Reader library to scale to read many SSTables without OOMing, and controls the flow by
  * buffering more bytes on-demand as the data is drained.
  * <p>
  * This class expects the consumer thread to be single-threaded, and the producer thread to be single-threaded OR serialized to ensure ordering of events.
  *
- * @param <T> SSTable
+ * @param <FileType> CassandraFile type e.g. SSTable, CommitLog
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
-public class SSTableInputStream<T extends SSTable> extends InputStream implements StreamConsumer
+public class BufferingInputStream<FileType extends CassandraFile> extends InputStream implements StreamConsumer
```

Comment:
Please either revert it back to `T` or use `F`. I think there are several other occurrences that the type parameter is named the same as class names. For the sake of readability, please update those and make the type parameter distinguishable.